sdccc/src/test/java/com/draeger/medical/sdccc/messages/TestMessageStorage.java

```diff
@@ -1864,8 +1867,7 @@ public void testDetermineCharsetFromMessageFromHttpHeader(@TempDir final File di
 
             // then
             assertEquals(Charset.forName("ISO-8859-13"), actualCharset);
-            Mockito.verify(this.testRunObserver, VerificationModeFactory.atLeastOnce())
-                    .invalidateTestRun(anyString());
+            verify(this.testRunObserver, VerificationModeFactory.atLeastOnce()).invalidateTestRun(anyString());
```

Comment:
what is the purpose of this change ? 

seems unnecessary

Reply:
Agreed, it is not necessary.
However, removing redundant information to make the code more readable is still best practice.

Reply:
How did it become more readable? If anything, it contains less information now.

Reply:
The information that "verify" is a static method from Mockito is redundant (it is also contained in the imports). Also, it is not neccessary as verify is not ambiguous.
Removing it at this place does hence not decrease the amount of information.
On the contrary, one has to read less code to gain all the relevant information, which is why I call it "more readable".

Reply:
Well, I do not agree with you, but I will resolve this conversation nontheless, because it is not important enough to let it block the pr.